Sources/TokamakDOM/Shapes/Path.swift

```diff
@@ -30,35 +30,85 @@ extension Path: ViewDeferredToRenderer {
                                                 dashPhase: 0)) -> AnyView {
     let stroke = [
       "stroke-width": "\(strokeStyle.lineWidth)",
-      "stroke": "black", // TODO: Use the environment variable "foregroundColor"
     ]
     let uniqueKeys = { (first: String, _: String) in first }
     switch storage {
     case .empty:
       return AnyView(EmptyView())
     case let .rect(rect):
-      return AnyView(AnyView(HTML("rect", ["width": "\(max(0, rect.size.width))",
-                                           "height": "\(max(0, rect.size.height))"]
-          .merging(stroke, uniquingKeysWith: uniqueKeys))))
-    case .ellipse:
-      return AnyView(HTML("ellipse", ["cx": "50%", "cy": "50%", "rx": "50%", "ry": "50%"]
+      return AnyView(AnyView(HTML("rect", [
+        "width": "\(max(0, rect.size.width))",
+        "height": "\(max(0, rect.size.height))",
+        "x": "\(rect.origin.x - (rect.size.width / 2))",
+        "y": "\(rect.origin.y - (rect.size.height / 2))",
+      ].merging(stroke, uniquingKeysWith: uniqueKeys))))
+    case let .ellipse(rect):
+      return AnyView(HTML("ellipse", ["cx": "\(rect.origin.x)",
+                                      "cy": "\(rect.origin.y)",
+                                      "rx": "\(rect.size.width)",
+                                      "ry": "\(rect.size.height)"]
           .merging(stroke, uniquingKeysWith: uniqueKeys)))
     case let .roundedRect(roundedRect):
       return AnyView(HTML("rect", [
         "width": "\(roundedRect.rect.size.width)",
         "height": "\(roundedRect.rect.size.height)",
         "rx": "\(roundedRect.cornerSize.width)",
-        "ry": "\(roundedRect.style == .continuous ? roundedRect.cornerSize.width : roundedRect.cornerSize.height)",
-      ]
-        .merging(stroke, uniquingKeysWith: uniqueKeys)))
+        "ry": """
+        \(roundedRect.style == .continuous ?
+          roundedRect.cornerSize.width :
+          roundedRect.cornerSize.height)
+        """,
+        "x": "\(roundedRect.rect.origin.x)",
+        "y": "\(roundedRect.rect.origin.y)",
+      ].merging(stroke, uniquingKeysWith: uniqueKeys)))
     case let .stroked(stroked):
-      return stroked.path.svgFrom(storage: stroked.path.storage, strokeStyle: stroked.style)
+      return AnyView(stroked.path.svgBody(strokeStyle: stroked.style))
     case let .trimmed(trimmed):
       return trimmed.path.svgFrom(storage: trimmed.path.storage, strokeStyle: strokeStyle) // TODO: Trim the path
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakDOM/Shapes/Path.swift;
  Line: 67;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 113 characters (`line_length`)